src/components/markers/HealthMarkers.tsx

```diff
@@ -23,32 +18,80 @@ type HealthMarkerProps = {
 }
 
 const HealthMarkers = ({ ageKey }: HealthMarkerProps) => {
+  //
+  const [expandCardio, setExpandCardio] = useState(false)
+  const [expandPhysical, setExpandPhysical] = useState(false)
+  const [expandFlexibility, setExpandFlexibility] = useState(false)
   //
   const results = healthData[ageKey]?.healthMarkers
 
-  console.log('This is results', results)
-
   return (
-    // <div className={styles.markersContainer}>
-    <div className="grid grid-cols-1 md:grid-cols-3 gap-2 items-center w-[100%]">
-      {results?.map((item, i) => {
-        return (
-          <div
-            className="flex flex-col justify-center items-center border-2 border-neutral rounded-xl p-5"
-            key={i}
-          >
-            <div className={styles.listHeader}>
-              <div style={{ fontSize: '40px' }}>
-                {iconMappings[`${item.type}`]}
-              </div>
+    <div className="w-[90%] bg-zinc-400 border-8 border-zinc-300">
+      <div className="flex flex-col justify-left items-center">
+        {' '}
+        <div className="flex justify-center items-center bg-gradient-to-r from-sky-700 to-blue-800 w-[100%] py-1 h-[70px] shadow-md shadow-zinc-600">
+          {' '}
+          <p className="text-white text-2xl md:text-4xl font-serif font-semibold pl-1">
+            HEALTH MARKERS
+          </p>
+        </div>
+        <div className="py-5 md:py-10 px-2 md:px-10">
+          <p className="text-center text-white text-md md:text-xl font-sans">
+            Our physical health primarily falls into three categories.
+            Understanding various markers within each helps us plan effective
+            fitness routines, push ourselves when necessary, and stay away from
+            injury.
+          </p>
+        </div>
+        <p className="text-center text-sm font-sans italic text-white">
+          Click any button to learn more
+        </p>
+      </div>
+      <div className="flex justify-center">
+        <div className="grid grid-cols-1 md:grid-cols-3 gap-2 px-2 w-[90%] mt-3 mb-10">
+          {results?.map((item, i) => {
+            return (
+              <div
+                onClick={() => {
+                  item.type === 'cardiovascular' ? (
+                    (setExpandCardio(true),
+                    setExpandPhysical(false),
+                    setExpandFlexibility(false))
+                  ) : item.type === 'physical' ? (
+                    (setExpandPhysical(true),
+                    setExpandCardio(false),
+                    setExpandFlexibility(false))
+                  ) : item.type === 'flexibility' ? (
+                    (setExpandFlexibility(true),
+                    setExpandCardio(false),
+                    setExpandPhysical(false))
+                  ) : (
```

Comment:
This is a lot of similar and repeated logic that gums up the onClick handler. We should refactor this to a `setModalState` method that accepts the `item.type` and handles state. Then this changes to `onClick={() => setModalState(item.type)` where we define `setModalState` at the top of the file, outside the HealthMarker component.